generators/app/templates/.env

```diff
@@ -1,4 +1,5 @@
-REACT_APP_TICTAC_PATH=public/games/tictactoe/
+REACT_APP_TICTAC_PATH=public/test2/tictactoe/
 REACT_APP_VERSION=$npm_package_version
+REACT_APP_PERMISSIONS=Append,Control,Read,Write
```

Comment:
Why is this part of the config? Don't these modes come from the components? They aren't app-specific but rather solid specific

Reply:
This is an app-specific it's not for a component I think this should be here

Reply:
Right, but I meant I thought there was a service/class that provided this list in the ACL library?

Reply:
these are the permissions that app needs to works it's not related to ACL library.